ponyville/sass/blocks/_style.scss

```diff
@@ -0,0 +1,4 @@
+@import "button/style";
+@import "paragraph/style";
+@import "navigation/style";
+@import "group/style";
```

Comment:
I wonder if this will grow to Seedlet levels when we want this theme's children to be able to customize all the rest of the blocks. This is the part that worries me the most.